python/py-exif/Portfile

```diff
@@ -1,31 +1,25 @@
-PortSystem 1.0
-PortGroup python 1.0
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
 
-name			py-exif
-version			1.0.2
-license			BSD
-platforms		darwin freebsd
-supported_archs	noarch
-maintainers		nomaintainer
-description		Python interface to the EXIF meta-data
-long_description	Exchangeable Image File Format for Digital Still \
-				Cameras is a meta-information tag that can be embedded \
-				in tiff or jpeg image files. py-exif is a Python \
-				interface to this data.
+github.setup        ianare exif-py 2.1.2
+name                py-exif
+license             BSD
+platforms           darwin
+supported_archs     noarch
+maintainers         nomaintainer
+description         Python interface to the EXIF meta-data
+long_description    Exchangeable Image File Format for Digital Still \
+                    Cameras is a meta-information tag that can be embedded \
+                    in tiff or jpeg image files. py-exif is a Python \
+                    interface to this data.
 
-homepage		https://sourceforge.net/projects/exif-py/
-master_sites	sourceforge:exif-py
-distname		EXIFpy_${version}
-checksums		md5 7df9f94e98b6ff5da872105a11069356 \
-			sha1 859411ec882fc297a4aa4f1368aa5c2a14688573 \
-			rmd160 26bbc5fbdd01b7c8cf472895f66a9f633b0510fd
+checksums           rmd160  c24bf810a72a7a176833b7e1d564349d316fe868 \
+                    sha256  53b0fa8e33071320ab1d8ae4713a7bea859ec166bc3774df9003b25ffea94b90 \
+                    size    32200
 
-python.versions	27
+python.versions     27 33 34 35 36 37
```

Comment:
Why add all of the old python 3 versions? Please just add py37.

Reply:
I was not aware older versions should not be added to portfiles. I will remove the older ones, however the project only acknowledges it has been tested on versions up to 3.5.

Reply:
We are actively removing old python sub-ports. Only the latest stable version should be added to existing ports. Of course if it does not work on python >3.5, only py35 should be added.